manifest.json

```diff
@@ -30,6 +30,21 @@
                 "name": "domain",
                 "type": "domain"
             },
+            {
+                "name": "admin",
+                "type": "user"
+            },
+            {
+                "name": "email",
+                "type": "email",
+                "ask": {
+                    "en": "Enter email address for administative notifications."
+                }
+            },
```

Comment:
To try to reiterate: the CI is failing because it needs a value to fill this question during the test. It should be defined in the file `check_process`

Or alternatively, as pointed in a previous comment, if the point is to obtain the email address of the `admin` user chosen in the previous question, you could fetch it using `email=$(ynh_user_get_info "$admin" 'mail')` inside the install script

Reply:
Yeah, I want to ask from the user specifically since the ynh_user_get_info one might not be functional with dns etc. At least it isn't on any of my instances. Will add it to the check_process and retest!